docs/source/introduction/makeyourown.rst

```diff
@@ -179,7 +179,8 @@ The boefje itself imports the shodan api module, assigns an IP address to it and
 Normalizers
 -----------
 
-The normalizer imports the raw information, extracts the objects from it and gives them to Octopoes. Since OpenKAT 1.3.0, the normalizers are fully self-contained. They consist of the following files:
+The normalizer imports the raw information, extracts the objects from it and gives them to Octopoes.
+Since OpenKAT 1.3.0, the normalizers are fully self-contained. They consist of the following files:
```

Comment:
I know that it was already there, but given that we are making changes, I don't think it is relevant anymore that something was changed in OpenKAT 1.3.

```suggestion
The normalizer imports the raw information, extracts the objects from it and gives them to Octopoes. They consist of the following files:
```